daemon/rpc_claim_online_purchase.go

```diff
@@ -0,0 +1,26 @@
+package daemon
+
+import (
+	"context"
+	"log"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+func (r *RPC) ClaimOnlinePurchase(ctx context.Context, in *pb.Empty) (*pb.Empty, error) {
+	isExpired, err := r.ac.IsVPNExpired()
+	if err != nil {
+		log.Println(internal.ErrorPrefix+" failed to determine if user is registered: ", err)
+	}
+
+	if isExpired {
+		log.Println(internal.DebugPrefix + " user is expired when caliming online purchase.")
+		return &pb.Empty{}, nil
+	}
+
+	log.Println(internal.DebugPrefix + " send user subscribed notification.")
```

Comment:
this might not be a very helpful message. could you add more info?

Reply:
What kind of info?

Reply:
what is not clear for me is if this is a message that we need to send a notification into the app(kind of a TODO), or if we log that the user subscription was updated.

if it is the second(we log that this function was executed), wouldn't it be better something like `updated user [subscription] plan`?